src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2791,19 +2817,30 @@ def add_cors(cmd, resource_group_name, name, allowed_origins, slot=None):
     if not configs.cors:
         configs.cors = CorsSettings()
     configs.cors.allowed_origins = (configs.cors.allowed_origins or []) + allowed_origins
-    result = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'update_configuration', slot, configs)
+    
+    result = None
+    if is_centauri_functionapp(cmd, resource_group_name, name):
+        result = update_configuration_polling(cmd, resource_group_name, name, slot, configs)
+    else:
+        result = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'update_configuration', slot, configs)
+    
     return result.cors
 
-
```

Comment:
Can you run `azdev style appservice`? I am pretty sure removing this space here will make the pipeline fail.